content/2022/videookulografija.md

```diff
@@ -27,30 +40,81 @@ Kako bi lakše detektovali ivice naše zenice prvo je potrebno RGB(crvena-zelena
 10. $B4=(B1+B2+B3)/3$
 11. $I1=(R4+G4+B4+UV)/4$
 
+
 U koracima od 1 do 3 izračunavamo osvetljenost i vrednosti hrominacije naše primarne slike u boji. RGB vrednosti su aproksimirane u koracima od 5 do 10 korišćenjem RGB komponenti. U 11. koraku izračunavanjem proseka vrednosti R4, G4, B4 i UV dobijamo vrednost I1 koja predstavlja našu sliku u sivom spektru. 
 
 #### **Zamućivanje**
 Kako bi naša detekcija ivice funkcionisala, neophodno je da sliku zamutimo Gausovim filterom i na taj način smanjimo količinu detalja na datoj slici. Gausov filter je zapravo niskopropusni filter  jer ima efekat smanjenja visokofrekventnih komponenti slike. Korišćenjem Gausove funkije za dve dimenzije dobijamo vrednosti iz kojih se gradi konvoluciona matrica koja se primenjuje na sliku sivog spektra. Nove vrednosti svakog piksela postavljaju se na srednje vrednosti susednih piksela. Vrenosti originalnog piksela dobija najveću Gausovu vrednost, susedni pikseli dobijaju manje vrednosti kako bi se njihova udaljenost od origanalnih piksela poveća. Na ovaj način se rezultira zamućenost kojom se čuvaju granice i ivce bolje nego drugim metodama.
 
 
+#### **Binarizacija**
+Binarizacija za cilj ima da izdvoji objekat iz slike, u ovom slučaju zenicu. Postiže se tako što sliku koju smo prebacili u sivu spektar sada konvertujemo u crno-belo. Prvo je potrebno naći trešhold vrednosti, odnosno vrednost praga, što radimo pomoću histograma. Kao trešhold uzimamo najveću vrednost kumulativne sume histograma. Sada dodeljujemo vrednosti pikselima na sledeći način – oni pikseli čija je vrednost manja ili jednaka od vrednosti trešholda, dobijaju vrednost 0 (postaju crni), a oni čija je vrednost veća od trešholda dobijaju vrednost 255 (postaju beli). Sada možemo da vidimo jasno izdvojenu zenicu (slika znice). 
```

Comment:
Kada izdvajamo objekat iz slike to zovemo segmentacija. Zadatak binarizacije je da za svaki piksel označi jednom od dve labele (0,1) u zavisnosti od kriterijuma koji postavimo sa ciljem da to bude prvi korak pri segementaciji. Nije spektar nego prostor boja. Trešhold se kaže prag. Kao threshold ne uzimamo najveću vrednost kumulativne sume histograma? Odakle vam to? Za Threshold biramo onu vrednost od koje je 17.5% piksela sa slike manje.